pkg/client/unversioned/request.go

```diff
@@ -841,8 +841,8 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 
 	// Did the server give us a status response?
 	isStatusResponse := false
-	var status *unversioned.Status
-	result, err := runtime.Decode(r.content.Codec, body)
+	status := &unversioned.Status{}
+	result, _, err := r.content.Codec.Decode(body, nil, status)
```

Comment:
Although, I guess we could say that this would be slightly better because it would capture all versions.  Hrm.. I guess the into is fine, but use `runtime.DecodeInto` instead of having the empty arguments.